### PR TITLE
feat: login ui

### PR DESCRIPTION
**Pull request by anto1313**
i have made ui login

### CODE REVIEW

File: src/app/(auth)/login/page.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Login } from "@/components/auth/components/Login";
+
+export default function Page() {
+  return <main>
```
Comment (indrazm) [inline src/app/(auth)/login/page.js:4]: You don't need <main>. Please remove it.

File: src/app/page.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+import { Login } from "@/components/auth/components/Login";
+
 export default function Page() {
-  return <main>Devwiz</main>;
+  return <main>
```
Comment (indrazm) [inline src/app/page.js:4]: You don't need <main>. Please remove it.

**Review by indrazm — CHANGES_REQUESTED** [review]
Good

File: src/app/(auth)/login/page.js (hunk 1 above)
Comment (anto1313) [reply src/app/(auth)/login/page.js:4]: remove the code or the file sir ?

File: src/app/(auth)/register/page.js
Comment (indrazm) [inline src/app/(auth)/register/page.js:1]: No need `<main>`

**Review by nurulid — APPROVED** [review]
Good

**Pull request merged**